spec/integration/create_subscriber_list_spec.rb

```diff
@@ -5,26 +5,17 @@
     end
 
     it "creates a subscriber_list" do
-      create_subscriber_list(tags: { topics: { any: ["oil-and-gas/licensing"] },
-                                     location: { all: %w[france germany] } })
-
+      create_subscriber_list
       expect(SubscriberList.count).to eq(1)
     end
 
     it "returns a 200" do
-      create_subscriber_list(tags: { topics: { any: ["oil-and-gas/licensing"] },
-                                     location: { all: %w[france germany] } })
-
+      create_subscriber_list
```

Comment:
I wonder if it's worth some of these tests being combined? We're doing the same thing multiple times, and then asserting on different aspects of the result. Seems like we could shave off some seconds in a test run and save the planet?

Reply:
Sure, I've done that. I had the same thought myself, but it seemed relatively minor at the time.